typescript/src/cairo/index.ts

```diff
@@ -0,0 +1,73 @@
+import dotenv from "dotenv";
+import fs from "fs";
+
+import type { ContractFactoryParams, RawArgs } from "starknet";
+import {
+  Account,
+  json,
+  RpcProvider,
+  CallData,
+  Contract,
+  ContractFactory,
+} from "starknet";
+
+dotenv.config();
+const ACCOUNT_ADDRESS = process.env.STARKNET_ACCOUNT_ADDRESS;
+const PRIVATE_KEY = process.env.STARKNET_PRIVATE_KEY;
+const CAIRO_BUILD_FOLDER = "../cairo/target/dev";
+
+/// Creates a Starknet account from the .env variables provided.
+export async function buildAccount(): Promise<Account> {
+  const provider = new RpcProvider({ nodeUrl: process.env.STARKNET_RPC_URL });
+
+  if (!PRIVATE_KEY || !ACCOUNT_ADDRESS) {
+    throw new Error("Private key or account address not set in .env file");
+  }
+  return new Account(provider, ACCOUNT_ADDRESS, PRIVATE_KEY);
+}
+
+/// Reads from a pre-compiled contract file.
+function getCompiledContract(name: string): any {
+  const prefix = name.includes("FeedsRegistry")
```

Comment:
this is a bit confusing, let's just assume the name is correct by default, wdyt ?

Reply:
done